.stylelintrc

```diff
@@ -19,7 +19,8 @@
     "custom-property-empty-line-before": "never",
     "no-eol-whitespace": null,
     "selector-nested-pattern": "^(&(:[\\w-]+|\\.state(Hover|Focus|FocusWithin|Active)),?\\s*)+$",
-    "max-nesting-depth": 1
+    "max-nesting-depth": 1,
+    "selector-class-pattern": null
```

Comment:
[stylelint-config-standard enforces kebab case](https://github.com/stylelint/stylelint-config-standard/blob/main/index.js#L129), which we don't care about rn